.github/workflows/ecs-deploy-v2.yml

```diff
@@ -90,27 +97,49 @@ jobs:
       - name: Terraform validate
         run: terraform validate
 
-      - name: Get current version
-        id: current_version
+      - name: Get former version from ECS
+        if: inputs.docker_image_tag || inputs.use_version_as_docker_image_tag
+        id: ecs_lookup
         run: |
           CURRENT_TASK_DEFINITION="$(aws ecs list-task-definitions | jq --arg SERVICE "$SERVICE" --arg ENVIRONMENT "$ENV" -r -c '.taskDefinitionArns[] | select(contains($SERVICE + "-" + $ENVIRONMENT))')"
           CURRENT_IMAGE_TAG="$(aws ecs describe-task-definition --task-definition "$CURRENT_TASK_DEFINITION" | jq --arg SERVICE "$SERVICE" -r -c '.taskDefinition.containerDefinitions[] | select(.name == $SERVICE) | .image' | cut -d':' -f2)"
           echo "image_tag=$CURRENT_IMAGE_TAG" >> "$GITHUB_OUTPUT"
 
+
+      - name: Get former version from Git
+        id: git_hash
+        if: "! (inputs.docker_image_tag || inputs.use_version_as_docker_image_tag)"
+        uses: sencrop/github-workflows/actions/look-git-hash@master
+
       - name: Notify deployment in progress
         uses: sencrop/github-workflows/actions/notify-deployment-in-progress@master
         with:
           service: ${{ inputs.service }}
           environment: ${{ inputs.environment }}
           dd_api_key: ${{ secrets.DD_API_KEY }}
-          current_version: ${{ steps.current_version.outputs.image_tag }}
-          deployed_version: ${{ inputs.docker_image_tag }}
+          former_version: ${{ steps.ecs_lookup.outputs.image_tag || steps.git_hash.output.previous }}
+          new_version: ${{ inputs.docker_image_tag || steps.git_hash.output.current }}
```

Comment:
Are the lint errors due to the linter having difficulty with understanding the output of the new workflow ?